sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1022,6 +1022,9 @@
               <argLine>
                 ${defaultSurefireArgLine}
                 ${javaModulesSurefireArgLine}
+                <!-- This is required as the Java 8 based implementation for ReflectionUtils may be loaded by IntelliJ when running tests with later versions of Java. -->
+                <!-- In later versions of Java, java.base doesn't open java.lang.invoke to azure-core, so it needs to be done with this directive. This only affects tests. -->
+                --add-opens java.base/java.lang.invoke=com.azure.core
```

Comment:
Should this be added in `azure-core`'s pom.xml instead of adding it in parent pom?

Reply:
Unfortunately, this is needed by every SDK that uses Core, so to me the best spot for it is in the parent POM